subseas_flex_fcst/maproom.py

```diff
@@ -314,42 +375,49 @@ def draw_colorbar(proba, variable, percentile):
     Input("proba", "value"),
     Input("variable", "value"),
     Input("percentile", "value"),
-    Input("threshold", "value")
+    Input("threshold", "value"),
+    Input("startDate","value"),
+    Input("leadTime","value")
 )
-def fcst_tile_url_callback(proba, variable, percentile, threshold):
+def fcst_tile_url_callback(proba, variable, percentile, threshold, startDate, leadTime):
 
     try:
         if variable != "Percentile":
             if threshold is None:
                 return "", True
             else:
-                return f"{TILE_PFX}/{{z}}/{{x}}/{{y}}/{proba}/{variable}/{percentile}/{float(threshold)}", False
+                return f"{TILE_PFX}/{{z}}/{{x}}/{{y}}/{proba}/{variable}/{percentile}/{float(threshold)}/{startDate}/{leadTime}", False
         else:
-            return f"{TILE_PFX}/{{z}}/{{x}}/{{y}}/{proba}/{variable}/{percentile}/0.0", False
+            return f"{TILE_PFX}/{{z}}/{{x}}/{{y}}/{proba}/{variable}/{percentile}/0.0/{startDate}/{leadTime}", False
     except:
         return "", True
 
 
 # Endpoints
 
-
 @SERVER.route(
-    f"{TILE_PFX}/<int:tz>/<int:tx>/<int:ty>/<proba>/<variable>/<float:percentile>/<float(signed=True):threshold>"
+    f"{TILE_PFX}/<int:tz>/<int:tx>/<int:ty>/<proba>/<variable>/<float:percentile>/<float(signed=True):threshold>/<startDate>/<leadTime>"
 )
-def fcst_tiles(tz, tx, ty, proba, variable, percentile, threshold):
-
+def fcst_tiles(tz, tx, ty, proba, variable, percentile, threshold, startDate,leadTime):
     # Reading
-
-    fcst_mu, fcst_var, obs, hcst = read_cptdataset(y_transform=CONFIG["y_transform"])
-
+    fcst_mu, fcst_var, dofVar, obs, hcst = read_cptdataset(leadTime, startDate, y_transform=CONFIG["y_transform"])
     # Get Issue date and Target season
-    # Hard coded for now as I am not sure how we are going to deal with time
-    issue_date = pd.to_datetime(["2022-04-01"])
-    target_start = pd.to_datetime(["2022-04-02"])
-    target_end = pd.to_datetime(["2022-04-08"])
+    issue_date_td = pd.to_datetime(startDate)
+    if leadTime == "Week 1":
+        lead_time = 1
```

Comment:
Are you sure this should be 1 (the day after the issue date) and not 0?

Reply:
I'm asking a question about the meaning of the original data, not about how we want to represent it internally (we're having that latter discussion on an email thread).

Reply:
The numbers I use here are based off the email Remi originally sent regarding S,L time and based off his original code from when he was only using a single data file where he specified:

issue_date = 2022-04-01 (start date)
target_start = 2022-04-02
target_end = 2022-04-08

with a lead time of "week 1" which I assumed would be adding a single day since target_start = issue_date +1